objcgen/objcgenerator-processor.cs

```diff
@@ -196,6 +202,14 @@ public override void Process (IEnumerable<Assembly> assemblies)
 					if (props.Count > 0)
 						properties.Add (t, props);
 
+					if (subscriptProps.Count > 0) {
+						if (subscriptProps.Count > 1) {
+							delayed.Add (ErrorHelper.CreateWarning (1041, $"Indexed properties on {t.Name} is not generated because multiple indexed properties not supported."));
+							subscriptProps.RemoveRange (1, subscriptProps.Count - 1);
```

Comment:
This is not stable fwict.

I'm fine just removing all subscript properties in this case. Let me do that.